ckanext/search_autocomplete/utils.py

```diff
@@ -48,16 +50,20 @@ def autocomplete_datasets(terms: List[str]) -> List[Suggestion]:
         if item not in combined
     ]
 
-    return [
-        Suggestion(
-            href=tk.h.url_for("dataset.read", id=item["name"]),
+    include_harvest = tk.asbool(tk.config.get(CONFIG_INCLUDE_HARVEST_PORTAL, DEFAULT_INCLUDE_HARVEST_PORTAL))
+    results = []
+    for item in combined + other[: _get_autocomplete_limit() - len(combined)]:
+        harvested = tk.h.get_pkg_dict_extra(item, "harvest_portal")
+        if not include_harvest and harvested is not None:
```

Comment:
If `include_harvest` is set to `False` and item has no `harvest_portal`, it won't be added to suggestions. Doesn't it mean that all local packages will be ignored as well, when `include_harvest` is disabled(because they have no `harvest_portal`)?